<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionSettings</class>
 <widget class="QWidget" name="ConnectionSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Connection Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Beam Current: String</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="hxpd_save">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hxpd_restore">
          <property name="text">
           <string>Restore</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="energy_restore">
          <property name="text">
           <string>Energy Restore</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Beam Current: Int</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QSpinBox" name="spinBox">
          <property name="minimum">
           <number>-999999999</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setlist">
          <property name="text">
           <string>Set List</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="setlist_errorLevel">
          <property name="minimum">
           <number>-999999999</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Beam Current: Double</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="minimum">
           <double>-999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="unitLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Errors:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item row="4" column="1">
       <widget class="QWidget" name="placeHolder" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="hxpd_label">
        <property name="text">
         <string>energy</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QWidget" name="energyHolder" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="hxpd_label_2">
        <property name="text">
         <string>slit gap</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QWidget" name="slitGapHolder" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="hxpd_label_3">
        <property name="text">
         <string>M4</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QWidget" name="m4Holder" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="hxpd_label_4">
        <property name="text">
         <string>M4 Inboard</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QWidget" name="m4InboardHolder" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="hxpd_label_5">
        <property name="text">
         <string>M4 Outboard</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QWidget" name="m4OutboardHolder" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="hxpd_label_6">
        <property name="text">
         <string>M4 Downstream</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QWidget" name="m4DownstreamHolder" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="hxpd_label_7">
        <property name="text">
         <string>Grating</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QWidget" name="gratingHolder" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
